### DISCUSSION TITLE
12주차 과제

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#45] [Show and tell]
#과제
1. Data Visualization 코드
2. uosgenerative.site/xxxx 에 자기 과제 파일 올려보기.

월요일 밤 12시까지.
더 일찍 제출하고 많이 질문할 수록 많은 도움을 받을 수 있음!
시간 늦게 제출하면 기껏 열심히 했는데 감점! ㅠ

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222594658387

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222594735766

**Comment by Joeun410** [comment]
김조은
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222594742206&categoryNo=27&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=7811

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye/222594753156

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222594861084

**Reply by jwejwepark** [reply]
박주혜
https://blog.naver.com/glqkfl0726/222595081855
